src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -125,6 +125,12 @@ def load_arguments(self, _):
         c.argument('key', help='If no key specified, return all keys by default. Support star sign as filters, for instance abc* means keys with abc as prefix. Similarly, *abc and *abc* are also supported.')
         c.argument('label', help="If no label specified, list all labels. Support star sign as filters, for instance abc* means labels with abc as prefix. Similarly, *abc and *abc* are also supported.")
 
+    with self.argument_context('appconfig kv restore') as c:
+        c.argument('name', id_part=None)
```

Comment:
add argument for yes.   
c.argument('yes', help="Do not prompt for preview.")

since for restore command arguments, "yes" has a different meaning with argument context ('appconfig').

Reply:
We currently use c.argument('yes', options_list=['--yes', '-y'], help='Do not prompt for confirmation.'). Does it really matter if it is preview vs confirmation? Because preview==confirmation. If you preview, you must confirm, too. You can only confirm, if you previewed.

Reply:
Sounds good.